### DISCUSSION TITLE
Home Screen Pages

### DISCUSSION THREAD

**Discussion by nightshepherdess** [TBog/TBLauncher#77] [Q&A]
Hello, I am completely new to this and have not used Kiss launcher. I came to this app simply because I despise having the Google search bar on my screen. But I do like having widgets present, just not on the home screen. I see in the options for widgets that you can move to a left or right screen, but I am not sure how to access or if I can access those screens. If not is there a way to add a second page to the home screen?

**Comment by TBog** [comment]
Enable `Settings` ➡ `Features` ➡ `Wallpaper interactions` ➡ `Scroll wallpaper`
Then you'll be able to scroll (drag the wallpaper) to the left and right page/screen

**Reply by HunterArmstrong** [reply]
Is there a way to make the "pages" scroll up and down instead? That would seem pretty cool.

**Reply by TBog** [reply]
Currently vertical scrolling is not possible.
1. I'd have to decide what to do with the widgets on the left and right screens if they are not accessible anymore.
2. Do you intend to scroll both vertically and horizontally at the same time?
3. Would you like to have widget screens scroll independent of the wallpapers?

**Reply by HunterArmstrong** [reply]
> Currently vertical scrolling is not possible.
> 1. I'd have to decide what to do with the widgets on the left and right screens if they are not accessible anymore.

I think that may require deleting the widgets before setting it or moving all the widgets to one general area for someone to move themself. But, I don't know. I do know that whatever you decide on will be good enough for the launcher to function well.

> 2. Do you intend to scroll both vertically and horizontally at the same time?

No, just vertically. 

> 3. Would you like to have widget screens scroll independent of the wallpapers? 

Yes. But I don't see why I would want to take away the option of the widget screen moving with the wallpaper away from other users. They may or may not have an interactive wallpaper.

**Reply by victor0d** [reply]
Sorry for reopening this topic. I am not really sure if I have missed anything, but currently I only manage to scroll the widgets together with the wallpaper (like in a big screen rather than individual screens). Is there any option to have the widgets screens (either vertical or horizontal) move in screens rather than progressive wallpaper drag?

**Reply by TBog** [reply]
> Is there any option to have the widgets screens move in screens rather than progressive wallpaper drag?

1. I don't understand how that would work. If you don't drag the screen, how do you expect to change the widget page? 

2. We are talking exclusively about the "Widget" desktop, right?

3. Did you try to enable both **Center wallpaper** and **Side stick wallpaper** options?

**Reply by victor0d** [reply]
My apologies TBog. You are so damn right. Setting the "Center" and "Side stick wallpaper" provides the expected behavior. I did missed. SORRY and THANKS.
